src/manifest/core.clj

```diff
@@ -1,15 +1,29 @@
 (ns manifest.core
   (:require [clojure.java.io :as io]
+            [clojure.string]
             [clojure.walk :as walk])
   (:import [java.util.jar Manifest]
            [java.net URL]))
 
 ;; Note that the manifest you want will only be on the classpath if it lives in the uberjar
 
+(defn- join-continuations [input]
+  (let [input (vec input)
+        continuation-indexes (keep-indexed #(when (= \space (first %2)) %1) input)
+        spliced (reduce (fn [v i]
+                          (update-in v [(dec i)] str
+                                     (clojure.string/replace-first (get v i) " " "")))
+                        input
+                        (reverse continuation-indexes))
+        continuation-indexes (set continuation-indexes)
+        spliced (keep-indexed (fn [i v] (when-not (continuation-indexes i) v)) spliced)]
+    spliced))
```

Comment:
the last `spliced` after the let you return is not really necessary you could replace it with the last line `(keep-indexed...)` so you don't need to shadow `spliced`.

in fact is not there a simplest solution of what you are doing here? I just realised you're shadowing `continuation-indexes` too, feels like a smell...

btw if you need this quick let me know and i just merge this and will come back to it some point ;)